docs/schema/PIDControlStatic.schema.json

```diff
@@ -49,12 +51,17 @@
             "type": [
                 "string"
             ]
+        },
+        "target_level": {
+            "format": "double",
+            "description": "target_level",
+            "type": "number"
         }
     },
     "required": [
         "node_id",
         "listen_node_id",
-        "proportional"
+        "target_level"
```

Comment:
Why the change from proportional to target_level? You've used coalesce elsewhere, so that implies it's not always required?

Reply:
It's not a change from one to the other. Proportional was removed as a required input so that they are all optional. This is handled nicely in the Julia core by skipping the computations for the terms where the coefficient is not defined (NaN). Target level is required as discussed above.